glmtools/tests/test_glm.py

```diff
@@ -6,7 +6,8 @@
 """
 
 import numpy as np
-
+import scipy.stats as stats
+import numpy.linalg as npl
```

Comment:
Generally put similar imports together, to help people to look for the imports they are expecting.  Here `numpy.linalg` should go up with `numpy`